src/Filesystem/Replicate.php

```diff
@@ -185,8 +183,10 @@ public function getMetadata($key)
      * Gets the class names as an array for both adapters.
      *
      * @return string[]
+     *
+     * @phpstan-return class-string[]
```

Comment:
Might be used in the `@return`.

Reply:
Not sure, vscode does not detect it, are you using it on admin bundle without the prefix?

Reply:
I found one `@param array<class-string, string[]> $adminClasses` and the rest is in the phpstan-return. So let's continue this way indeed. I'm too much phpstorm-oriented.